suzieq/cli/sqcmds/command.py

```diff
@@ -332,8 +332,13 @@ def _gen_output(self, df: pd.DataFrame, json_orient: str = "records",
                     if is_error:
                         print(df[cols])
                     else:
-                        sort_fields = [x for x in self.sqobj.sort_fields
-                                       if x in df.columns and x in cols]
+                        if (((self.start_time and self.end_time) or
+                             self.view == "all") and
+                                'timestamp' in df.columns):
+                            sort_fields = ['timestamp']
+                        else:
+                            sort_fields = [x for x in self.sqobj.sort_fields
+                                           if x in df.columns and x in cols]
```

Comment:
Just a general comment, probably unrelated specifically to this PR. Shouldn't we do the ordering in the engine instead of the CLI? As it is now we may have different results for the same commands in the CLI vs the rest server since the final ordering happens here.

Reply:
In general, I leave ordering to be a front-end thingy since with a lot of data, ordering takes time.